packages/iov-encoding/src/encoding.spec.ts

```diff
@@ -313,3 +313,21 @@ describe("Encoding", () => {
     });
   });
 });
+
+describe("Bech32", () => {
+  const lskAddressUtf8 = Encoding.toUtf8("6472030874529564639L");
+  const riseAddressUtf8 = Encoding.toUtf8("6472030874529564639R");
```

Comment:
The Lisk and RISE tests are irritating because there is no proper binary representation for those, so using them as example data is not very helpful. We will also not use Bech32 to encode Lisk and RISE addresses